### DISCUSSION TITLE
Automatic note numbering

### DISCUSSION THREAD

**Discussion by AndyCoach** [TriliumNext/Trilium#3275] [Ideas]
Can there be a feature where, once there are notes, sub-notes, sub-sub-notes, and so on, we could assign note/sub-note numbers (in order) automatically? For example

1.0 Cookies
2.0 Cakes
2.1 Chocolate cakes
2.2 Cheesecakes
2.2.1 Cheesecakes with cookies
2.2.1.1 Recipes
2.2.1.1.1 Recipe A
...

**Comment by AndyCoach** [comment]
No interest?

**Reply by zadam** [reply]
Hi, to be completely honest, I don't see the benefit ...

**Comment by AndyCoach** [comment]
Hi, it would be amazing to have it for people like myself, who have to create and manage enormous engineering (or other) Specifications documents. Such docs must have this type of numbering and tend to have hundreds of pages. People use Word and similar tools to create these specs documents but if a doc needs to evolve and change, the word processor solution fails miserably, it is just too hard to maintain such a beast in Word. I used to use TreePad for this but it was a 100% a desktop tool, no real time collaboration. I would be happy to make a donation if you would consider adding this feature.. or maybe writing script for Trilio that does this?
Cheers, - Andrew


Thursday, November 3, 2022, 3:42:43 PM, you wrote:

> Hi, to be completely honest, I don't see the benefit ...
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.

-- 
Best regards,
Andy
